### PR TITLE
Added full-stack dev inputs

### PR DESCRIPTION
**Pull request by sixthcodebrewer**
#247 What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Enhancement
- [ ] Documentation Update

## Description
added field for tech-stack of the developer, options like MEAN, MERN, JAM, etc.
## Related Tickets & Documents
#247 

## QA Instructions, Screenshots, Recordings

- [image](https://github.com/sixthcodebrewer/fullstack-svg/blob/main/Screenshot%20from%202020-10-10%2010-12-05.png?raw=true)

- The deployed version

### CODE REVIEW

File: src/constants/skills.js [hunk 1]
```diff
@@ -410,7 +423,16 @@ const icons = {
   vuepress:
     "https://raw.githubusercontent.com/AliasIO/wappalyzer/master/src/drivers/webextension/images/icons/VuePress.svg",
   unity:
-    "https://www.vectorlogo.zone/logos/unity3d/unity3d-icon.svg"
+    "https://www.vectorlogo.zone/logos/unity3d/unity3d-icon.svg",
+  JAM_stack: 
+    "https://raw.githubusercontent.com/sixthcodebrewer/fullstack-svg/be9264224dc17677a90b1937610264a587df5585/jam.svg",
+  MEAN_stack:
+   "https://raw.githubusercontent.com/sixthcodebrewer/fullstack-svg/be9264224dc17677a90b1937610264a587df5585/mean.svg",
```
Comment (aravindvnair99) [inline src/constants/skills.js:430]:
```suggestion
   "https://meanjs.org/img/logo.png",
```

File: src/constants/skills.js [hunk 2]
```diff
@@ -410,7 +423,16 @@ const icons = {
...
+  MERN_stack:
+    "https://raw.githubusercontent.com/sixthcodebrewer/fullstack-svg/be9264224dc17677a90b1937610264a587df5585/mern.svg",
```
Comment (aravindvnair99) [inline src/constants/skills.js:432]:
```suggestion
    "https://upload.wikimedia.org/wikipedia/commons/9/94/MERN-logo.png",
```

File: src/constants/skills.js [hunk 3]
```diff
@@ -410,7 +423,16 @@ const icons = {
   vuepress:
     "https://raw.githubusercontent.com/AliasIO/wappalyzer/master/src/drivers/webextension/images/icons/VuePress.svg",
   unity:
-    "https://www.vectorlogo.zone/logos/unity3d/unity3d-icon.svg"
+    "https://www.vectorlogo.zone/logos/unity3d/unity3d-icon.svg",
+  JAM_stack: 
+    "https://raw.githubusercontent.com/sixthcodebrewer/fullstack-svg/be9264224dc17677a90b1937610264a587df5585/jam.svg",
```
Comment (aravindvnair99) [inline src/constants/skills.js:428]:
```suggestion
    "https://raw.githubusercontent.com/jamstack/jamstack.org/main/src/site/img/logo/svg/Jamstack_Icon_Original.svg",
```

File: src/constants/skills.js (hunk 2 above)
Comment (sixthcodebrewer) [reply src/constants/skills.js:432]: hey i guess only svg are allowed !

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@sixthcodebrewer @rahuldkjain Isn't it better to use the official links or links that won't change rather than the repo of @sixthcodebrewer? Because now if the repo username or URL changes, that can cause issues. Else commit in the logo as well.

File: src/constants/skills.js (hunk 2 above)
Comment (aravindvnair99) [reply src/constants/skills.js:432]: @sixthcodebrewer Ahh my bad, better to commit in the logo as well I feel.

**Comment by sixthcodebrewer** [comment]
> @sixthcodebrewer @rahuldkjain Isn't it better to use the official links or links that won't change rather than the repo of @sixthcodebrewer? Because now if the repo username or URL changes, that can cause issues. Else commit in the logo as well.

we have found an alternative for the same. Well, your changes are against hack October fest rules.! Thank you for your feedback.

File: src/constants/skills.js [hunk 4]
```diff
@@ -536,7 +556,12 @@ const icons = {
   postman: postman,
   erlang: erlang,
   sqlite: sqlite,
-  middleman: middleman
+  middleman: middleman,
+  JAM_stack: jamStack,
+  MEAN_stack: meanStack,
+  MERN_stack: mernStack,
+  LAMP_stack: lampStack,
```
Comment (Aditya-ds-1806) [inline src/constants/skills.js:563]: remove the comma here, after lampStack, that is why the build i failing i think

**Review by Aditya-ds-1806 — CHANGES_REQUESTED** [review]
Just remove the trailing comma, build will succeed

File: src/images/icons/fullstack/Jamstack_Icon_Original.svg [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<svg width="132" height="147" viewBox="0 0 132 147" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (harshcut) [inline src/images/icons/fullstack/Jamstack_Icon_Original.svg:1]: Keep the filename `jamstack.svg` for this file.

File: src/constants/skills.js (hunk 4 above)
Comment (sixthcodebrewer) [reply src/constants/skills.js]: No it didn't affect but got the bug. Thank you @Aditya-ds-1806

File: src/images/icons/fullstack/Jamstack_Icon_Original.svg (hunk 5 above)
Comment (sixthcodebrewer) [reply src/images/icons/fullstack/Jamstack_Icon_Original.svg:1]: yeah got it, fill=none. Thank you

File: src/constants/skills.js [hunk 6]
```diff
@@ -201,6 +210,16 @@ const categorizedSkills = {
     ],
   },
 
+  fullstack_dev: {
+    title: "Fullstack Development",
+    skills: [
+      "MERN_stack",
+      "MEAN_stack",
+      "LAMP_stack",
+      "JAM_stack",
```
Comment (harshcut) [inline src/constants/skills.js:219]: Change these to lowercase as all the other skills are following the same style.

File: src/constants/skills.js [hunk 7]
```diff
@@ -20,6 +21,14 @@ import scala from "../images/icons/ProgrammingLanguages/scala.svg";
 import swift from "../images/icons/ProgrammingLanguages/swift.svg";
 import typescript from "../images/icons/ProgrammingLanguages/typescript.svg";
 
+// Fullstack Development
+import jamStack from "../images/icons/fullstack/Jamstack_Icon_Original.svg";
+import mernStack from "../images/icons/fullstack/mern.svg";
+import meanStack from "../images/icons/fullstack/mean.svg";
+import lampStack from "../images/icons/fullstack/lamp.svg";
```
Comment (harshcut) [inline src/constants/skills.js:28]: make all import variables lowercase

File: src/images/icons/fullstack/Jamstack_Icon_Original.svg (hunk 5 above)
Comment (harshcut) [reply src/images/icons/fullstack/Jamstack_Icon_Original.svg:1]: You misunderstood. Change the filename from `Jamstack_Icon_Original.svg` to `jamstack.svg`.

File: src/constants/skills.js (hunk 7 above)
Comment (sixthcodebrewer) [reply src/constants/skills.js:28]: Sure 👍

File: src/constants/skills.js (hunk 4 above)
Comment (Aditya-ds-1806) [reply src/constants/skills.js]: oh okay😅

File: src/constants/skills.js (hunk 6 above)
Comment (sixthcodebrewer) [reply src/constants/skills.js:219]: all the changes are done.

**Comment by sixthcodebrewer** [comment]
@rahuldkjain could you review this pr please.

**Comment by harshcut** [comment]
We are keeping all PRs related to SVGs on hold for now.

**Comment by harshcut** [comment]
Closing the Pull Request due to conflicting changes.
Usage of locally hosted icons has been deprecated. For more information visit issue #278.

issue: [use of local SVG icons made README.md look bad](https://github.com/rahuldkjain/github-profile-readme-generator/issues/278)

**Pull request closed without merging**